PySimpleGUI.py

```diff
@@ -3543,6 +3547,22 @@ def SetIcon(self, icon=None, pngbase64=None):
         except:
             pass
 
+    def SetMinSize(self, size=None):
```

Comment:
Throughout the code, when there is a size parameter that is a tuple, it is written as (None, None), not None.  This signals to the caller that a tuple is expected.

Reply:
You're missing the point here.  I'm not interested in arguing which form is better, more readable, more accepted. 
I'm going to pass on these changes.  I tried to explain that the code differed from the rest of the code base and the documentation that's published and in use.  I'm not interested in incorporating code that differs from established norms.

Reply:
Take it easy. I am not arguing about anything. Just wanted to explain that the `size` in (None, None) as per your established norms. You probably add these features anyway in the future. It's up to you to appreciate other people's work for your project.